artifacts/deployment/etcdproxy-controller.yaml

```diff
@@ -0,0 +1,158 @@
+---
+# The Namespace where controller and controller resources are deployed.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-apiserver-storage
+spec:
+---
+# The ServiceAccount used by EtcdProxyController.
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: etcdproxy-controller-sa
+  namespace: kube-apiserver-storage
+---
+# The ServiceAccount used by EtcdProxy pods.
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: etcdproxy-sa
+  namespace: kube-apiserver-storage
+---
+# ClusterRole for etcdproxy-controller-sa to update status on EtcdStorage.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: etcdproxy-crd-clusterrole
+rules:
+- apiGroups: ["etcd.xmudrii.com"]
+  resources: ["etcdstorages"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["etcd.xmudrii.com"]
+  resources: ["etcdstorages/status"]
+  verbs: ["get", "watch", "list", "create", "update", "patch"]
```

Comment:
seems like this should just be update and patch

Reply:
I think we could need `get` and `list` because of [this line](https://github.com/xmudrii/etcdproxy-controller/blob/master/pkg/controller/etcdproxy/etcdproxy_controller.go#L298), but I have to verify.

Reply:
Fixed.